lib/elixir_sense/core/metadata.ex

```diff
@@ -12,6 +12,7 @@ defmodule ElixirSense.Core.Metadata do
             lines_to_env: %{},
             calls: %{},
             vars_info_per_scope_id: %{},
+            mods_funs: %{},
```

Comment:
Is it a good idea to merge this prop with `mods_funs_to_positions`?

Reply:
Yeah, we could have the position in the `ModFunInfo` struct. However, I think `mods_funs_to_positions` always keep an entry for `{mod, func, nil}` as a fallback for cases where we cannot introspect the arity. Ideally, we should start using the new `Metadata.get_call_arity/3` (see example [here](https://github.com/elixir-lsp/elixir_sense/blob/ba2356078979ec65f40c236ce527418a9fefd613/lib/elixir_sense.ex#L345)). It works for both functions and types, so we could improve "Go to definition" accuracy.